gp-maven-plugin/src/main/java/com/ibm/g11n/pipeline/maven/GPDownloadMojo.java

```diff
@@ -154,11 +154,17 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
     private void exportLanguageResource(ServiceClient client, SourceBundleFile bf, String language,
             File outBaseDir, OutputContentOption outContntOpt, BundleLayout bundleLayout,
-            LanguageIdStyle langIdStyle, Map<String, String> langMap)
+            LanguageIdStyle langIdStyle, Map<String, String> langMap, String srcLang)
             throws MojoFailureException {
         String srcFileName = bf.getFile().getName();
         String relPath = bf.getRelativePath();
-
+        
+        // truncate source suffix from target resource files
+        int extensionIndex = srcFileName.lastIndexOf('.');
+		String extension = (extensionIndex > 0) ? srcFileName.substring(extensionIndex) : "";
+		int srcSuffixIndex = (langMap.containsKey(srcLang)) ? srcFileName.lastIndexOf("_" + langMap.get(srcLang)) : srcFileName.lastIndexOf("_" + srcLang);
```

Comment:
srcLang or langMap.get(srcLang) returns BCP 47 language code. You need to check langIdStyle here. This line should be replaced with

int srcSuffixIndex = srcFileName.lastIndexOf("_" + getLanguageId(srcLang, langIdStyle, langMap));